compatibility-verifier/compCheck.sh

```diff
@@ -169,6 +182,11 @@ COMPAT_TESTER_PATH="pinot-integration-tests/target/pinot-integration-tests-pkg/b
 # create subdirectories for given commits
 workingDir=$1
 testSuiteDir=$(absPath "$2")
+
+BROKER_CONF=${testSuiteDir}/config/BrokerConfig.conf
```

Comment:
You will now be able to reuse ${BROKER_CONF} everywhere in the script.

Reply:
done!